README.md

```diff
@@ -1,4 +1,4 @@
-# SuperTest
+# [SuperTest](https://visionmedia.github.io/superagent/)
```

Comment:
@titanism, you might also want to correct this now that it has been moved:
~~https://visionmedia.github.io/superagent/~~ https://ladjs.github.io/superagent/